src/player.cpp

```diff
@@ -5568,7 +5568,10 @@ void player::update_needs( int rate_multiplier )
     if( has_trait("PLANTSKIN") ) {
         thirst_rate -= 0.2f;
     }
-
+    if( is_wearing("stillsuit") ) {
```

Comment:
It would probably be better to make this check for an item flag than a single item.